### PR TITLE
[deps](FE)Upgrade the velocity version that hive-exec depends on to 2.3

### PR DESCRIPTION
**Pull request by hf200012**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [x] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [x] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [x] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 36.64 seconds
 load time: 465 seconds
 storage size: 17123356910 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221214044258_clickbench_pr_63046.html

File: fe/fe-core/pom.xml [hunk 1]
```diff
@@ -613,7 +613,18 @@ under the License.
...
             <classifier>core</classifier>
             <scope>provided</scope>
-        </dependency>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.velocity</groupId>
+                    <artifactId>velocity</artifactId>
+                </exclusion>
+            </exclusions>
+    </dependency>
```
Comment (morningman) [inline fe/fe-core/pom.xml:622]: ident

**Review by morningman — COMMENTED** [review]
missing dist/LICENSE.txt

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**